tests/unit/cartography/intel/github/test_github.py

```diff
@@ -16,11 +16,13 @@
 from tests.data.github.rate_limit import RATE_LIMIT_RESPONSE_JSON
 
 
+@patch('cartography.intel.github.util.time.sleep')
```

Comment:
Unrelated to this change, but this was wrong for a while and added 2 extra seconds to each test unncessarily